code/modules/reagents/chemistry/recipes/slime_extracts.dm

```diff
@@ -627,6 +661,22 @@
 	if(P)
 		P.loc = get_turf(holder.my_atom)
 
+
+/datum/chemical_reaction/slimecrayon
+	name = "Slime Crayon"
+	id = "s_crayon"
+	required_reagents = list("blood" = 1)
+	required_container = /obj/item/slime_extract/pyrite
+	required_other = 1
+
+/datum/chemical_reaction/slimecrayon/on_reaction(datum/reagents/holder)
+	feedback_add_details("slime_cores_used","[type]")
+	var/list/crayons = subtypesof(/obj/item/toy/crayon)
+	var/chosen = pick(crayons)
```

Comment:
this has a chance to create a spraycan iirc. Should probably uses a weighted_pick() list.